debian/bionic/foreman/rules

```diff
@@ -16,6 +16,7 @@ build:
 	/bin/rm -f bundler.d/test.rb bundler.d/development.rb
 	/bin/sed -i '2i  gem "puma-status"' bundler.d/service.rb
 	BUNDLE_BUILD__SASSC=--disable-march-tune-native BUNDLE_RETRY=5 BUNDLE_JOBS=4 /usr/bin/bundle package
+	/usr/bin/python script/filter-package-json.py
```

Comment:
It's not executable with a shebang?

Reply:
It's not (and the shebang is using py3, which we don't install by default -- the script works with py2 and 3 identically)